src/Sylius/Component/User/Security/Checker/EnabledUserChecker.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\User\Security\Checker;
+
+use Sylius\Resource\Model\ToggleableInterface;
+use Symfony\Component\Security\Core\Exception\DisabledException;
+use Symfony\Component\Security\Core\User\UserCheckerInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+final class EnabledUserChecker implements UserCheckerInterface
+{
+    public function checkPreAuth(UserInterface $user): void
+    {
+        if (!$user instanceof ToggleableInterface) {
+            return;
+        }
```

Comment:
```suggestion
use Symfony\Component\Security\Core\User\UserInterface as SymfonyUserInterface;

final class EnabledUserChecker implements UserCheckerInterface
{
    public function checkPreAuth(UserInterface $user): void
    {
        if (!$user instanceof UserInterface) {
            return;
        }
```
It should only work on sylius user instances